packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -7,17 +7,19 @@ const { i18n } = require('@hubspot/cli-lib/lib/lang');
 
 const i18nKey = 'cli.lib.prompts.createProjectPrompt';
 
-const createTemplateOptions = async () => {
+const createTemplateOptions = async repoPath => {
+  const isRepoPath = !!repoPath;
   const config = await fetchJsonFromRepository(
-    'hubspot-project-components',
-    'main/config.json'
+    repoPath,
+    'main/config.json',
+    isRepoPath
   );
 
   return config[PROJECT_COMPONENT_TYPES.PROJECTS];
```

Comment:
not a blocker, but we could potentially do an additional error check here to make sure that the `config.json` in the repo has the right structure. I'm thinking about the scenario where the user puts a config.json in their repo, but it doesn't contain a `projects` field in it. Or if we want to go deeper, the values in the projects array don't contain the right fields. examples:

```js
const badConfig1 = {
  hubspot-projects: []
}

const badConfig2 = {
  projects: [
    { blah: ''}
  ]
}

```

Reply:
I don't think it should be too hard to add some basic checks.

Reply:
Added both checks in https://github.com/HubSpot/hubspot-cli/pull/837/commits/7d3c09c254666f4477d60bd043e23abf18360df0